tests/v2/validation/Jenkinsfile_support_matrix_os

```diff
@@ -0,0 +1,198 @@
+                      extensions: scm.extensions + [[$class: 'CleanCheckout']],
+                      userRemoteConfigs: scm.userRemoteConfigs
+                    ])
+          }
+          dir ("./") {
+              stage('Configure and Build') {
+                if (env.AWS_SSH_PEM_KEY && env.AWS_SSH_KEY_NAME) {
+                  dir("./tests/v2/validation/.ssh") {
+                    def decoded = new String(AWS_SSH_PEM_KEY.decodeBase64())
+                    writeFile file: AWS_SSH_KEY_NAME, text: decoded
+                  }
+                }
+
+                sh "./tests/v2/validation/configure.sh"
+
+              }
+              stage('Setup') {
+                sh returnStdout: true, script: 'wget -qO ./yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64'
+                sh returnStdout: true, script: 'chmod a+x ./yq'
+
+                rke1NodeParams = [
+                  string(name: 'TIMEOUT', value: "${env.TIMEOUT}"),
+                  string(name: 'CONFIG', value: "${env.RANCHER_RKE1_NODE_CONFIG}"),
+                  string(name: 'TEST_PACKAGE', value: "${env.RANCHER_RKE1_TEST_PACKAGE}"),
+                  string(name: 'GOTEST_TESTCASE', value: "${env.RANCHER_RKE1_NODE_GOTEST_TESTCASE}"),
+                ]
+                rke1CustomParams = [
+                  string(name: 'TIMEOUT', value: "${env.TIMEOUT}"),
+                  string(name: 'CONFIG', value: "${env.RANCHER_RKE1_CUSTOM_CONFIG}"),
+                  string(name: 'TEST_PACKAGE', value: "${env.RANCHER_RKE1_TEST_PACKAGE}"),
+                  string(name: 'GOTEST_TESTCASE', value: "${env.RANCHER_RKE1_CUSTOM_GOTEST_TESTCASE}"),
+                ]
+                rke2NodeParams = [
+                  string(name: 'TIMEOUT', value: "${env.TIMEOUT}"),
+                  string(name: 'CONFIG', value: "${env.RANCHER_RKE2_NODE_CONFIG}"),
+                  string(name: 'TEST_PACKAGE', value: "${env.RANCHER_RKE2_TEST_PACKAGE}"),
+                  string(name: 'GOTEST_TESTCASE', value: "${env.RANCHER_RKE2_NODE_GOTEST_TESTCASE}"),
+                ]
+                rke2CustomParams = [
+                  string(name: 'TIMEOUT', value: "${env.TIMEOUT}"),
+                  string(name: 'CONFIG', value: "${env.RANCHER_RKE2_CUSTOM_CONFIG}"),
+                  string(name: 'TEST_PACKAGE', value: "${env.RANCHER_RKE2_TEST_PACKAGE}"),
+                  string(name: 'GOTEST_TESTCASE', value: "${env.RANCHER_RKE2_CUSTOM_GOTEST_TESTCASE}"),
+                ]
+              } 
+              stage('Run Jobs') {
+                def amis = env.RANCHER_AMI_LIST
+                amiList = amis.split(',').each { obj -> 
+                String ami
+                String sshUser
+                if (obj.contains(':')){
+                  def data = obj.split(':')
+                  ami = data[0]
+                  sshUser = data[1]
+                } else {
+                  ami = obj
+                  sshUser = 'ec2-user'
```

Comment:
So if an ssh user isn't provided it defaults to ec2-user? Could you explain it please.

Reply:
The majority of common Linux OS AMIs either require or will accept `ec2-user` as the default username, so I figured it made sense to make it the default, for instance of the OS listed in the AWS docs, only Debian, Ubuntu and Bitnami require a non `ec2-user` default username. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connection-prereqs.html#connection-prereqs-get-info-about-instance

Reply:
works for me